api.planx.uk/modules/send/bops/bops.ts

```diff
@@ -69,7 +62,7 @@ const sendToBOPS = async (req: Request, res: Response, next: NextFunction) => {
       adapter: "http",
       headers: {
         "Content-Type": "application/json",
-        Authorization: `Bearer ${process.env.BOPS_API_TOKEN}`,
+        Authorization: `Bearer ${bopsToken || process.env.BOPS_API_TOKEN}`,
```

Comment:
This fallback can be removed once BOPS deprecate the old key on staging and production. We can then add a check that this is set before submitting.